profiler/options.go

```diff
@@ -71,6 +74,19 @@ func urlForSite(site string) (string, error) {
 	return u, err
 }
 
+// Check for API key constraints:
+func apiKeyCheck(key string) bool {
+	if optionAPIKeyLength != len(key) {
```

Comment:
Looks backwards :)
```suggestion
	if len(key) != 32 {
```

Reply:
Good catch, bad habit!

Reply:
It would be great if you could only resolve a comment thread after it was addressed, or allow the reviewer to resolve it.